### PR TITLE
add Community Layouts for `LAYOUT` keyboards

### PR DESCRIPTION
**Pull request by dunk2k**
## Description

Assign matching community_layout(s) to keyboards with `LAYOUT` macro
- where applicable; utilise `layout_aliases` to match layout name to Community Layout name

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — CHANGES_REQUESTED** [review]
These should ideally be swapped around, there is no functional change.

**Comment by dunk2k** [comment]
> These should ideally be swapped around, there is no functional change.

Apologies, could you elaborate on "swapped around"?

**Comment by fauxpark** [comment]
Layout aliases where the alias is a community layout name and the target is not should be swapped (as the idea is to remove the aliases at some later point). Such as:
```json
    "layout_aliases": {
        "LAYOUT_60_ansi": "LAYOUT_ansi_61"
    },
```

**Comment by noroadsleft** [comment]
Been looking through this, and most of it so far - I've reviewed not quite 1/4th of the files - is good.

Only thing I disagree with so far is I don't think the `cxt_studio` should be included in this. It's currently implemented as a 4x4 macropad, but it's actually a 3x4 macropad with 4 rotary encoders (available [here](https://www.aliexpress.us/item/3256804706946033.html), as well as a few other listings). I did an edit to this board a while back, but didn't change the layout because I wasn't able to definitively determine which keycode positions map to each encoders.

**Comment by dunk2k** [comment]
> Only thing I disagree with so far is I don't think the `cxt_studio` should be included in this. It's currently implemented as a 4x4 macropad, but it's actually a 3x4 macropad with 4 rotary encoders.

`cxt_studio` has, happily, been reverted 🙂

File: keyboards/knops/mini/keymaps/default-gsm-newbs/keymap.c [hunk 1]
```diff
@@ -9,7 +9,7 @@ enum custom_keycodes {
 *  Copy of knopps mini default May 16,2018
 *  Added comments in code to more easilly understand it.
 *
-*   Key Layout
+*   Key LAYOUT_ortho_2x3
```
Comment (noroadsleft) [inline keyboards/knops/mini/keymaps/default-gsm-newbs/keymap.c:12]:
```suggestion
*   Key Layout
```

File: keyboards/knops/mini/keymaps/default-gsm-newbs/keymap.c [hunk 2]
```diff
@@ -19,7 +19,7 @@ enum custom_keycodes {
 *	|  4  | |  5  | |  6  |
 *	|_____| |_____| |_____|
 *
-*  Each Layout row below keys. 1,2,3,4,5,6
+*  Each LAYOUT_ortho_2x3 row below keys. 1,2,3,4,5,6
```
Comment (noroadsleft) [inline keyboards/knops/mini/keymaps/default-gsm-newbs/keymap.c:22]:
```suggestion
*  Each Layout row below keys. 1,2,3,4,5,6
```

File: keyboards/knops/mini/keymaps/default-gsm-newbs/keymap.c [hunk 3]
```diff
@@ -69,11 +69,11 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 *	|_____| |_____| |_____|
 *
 */
-	LAYOUT(
+	LAYOUT_ortho_2x3(
 		LT(3, KC_1), KC_2, KC_3, KC_4, M_TGLHF, M_TGG),
 
 /*
-*  Layer 3 Key Layout
+*  Layer 3 Key LAYOUT_ortho_2x3
```
Comment (noroadsleft) [inline keyboards/knops/mini/keymaps/default-gsm-newbs/keymap.c:76]:
```suggestion
*  Layer 3 Key Layout
```

File: keyboards/knops/mini/keymaps/mverteuil/keymap.c [hunk 4]
```diff
@@ -53,7 +53,7 @@ tap_dance_action_t tap_dance_actions[] = {
 };
 
 /*
- *   Key Layout
+ *   Key LAYOUT_ortho_2x3
```
Comment (noroadsleft) [inline keyboards/knops/mini/keymaps/mverteuil/keymap.c:56]:
```suggestion
 *   Key Layout
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Thoughts at this point:

- `free_willy` isn't a split keyboard; it's a 36-key ortholinear with one controller
- `handwired/gamenum` key order isn't accurate to `numpad_5x4` community layout
- `handwired/jn68m`'s `LAYOUT_all` is equivalent to `LAYOUT_68_ansi_split_bs`

File: keyboards/lets_split/rev1/info.json [hunk 5]
```diff
@@ -23,6 +23,7 @@
     "layout_aliases": {
         "LAYOUT": "LAYOUT_ortho_4x12"
     },
+    "community_layout": ["ortho_4x12"],
```
Comment (fauxpark) [inline keyboards/lets_split/rev1/info.json:26]:
```suggestion
    "community_layouts": ["ortho_4x12"],
```

File: keyboards/lets_split/rev2/info.json [hunk 6]
```diff
@@ -23,6 +23,7 @@
     "layout_aliases": {
         "LAYOUT": "LAYOUT_ortho_4x12"
     },
+    "community_layout": ["ortho_4x12"],
```
Comment (fauxpark) [inline keyboards/lets_split/rev2/info.json:26]:
```suggestion
    "community_layouts": ["ortho_4x12"],
```

File: keyboards/lets_split/sockets/info.json [hunk 7]
```diff
@@ -23,6 +23,7 @@
     "layout_aliases": {
         "LAYOUT": "LAYOUT_ortho_4x12"
     },
+    "community_layout": ["ortho_4x12"],
```
Comment (fauxpark) [inline keyboards/lets_split/sockets/info.json:26]:
```suggestion
    "community_layouts": ["ortho_4x12"],
```

**Review by fauxpark — APPROVED** [review]
Might be more that I missed.

File: keyboards/lets_split/rev1/info.json (hunk 5 above)
Comment (dunk2k) [reply keyboards/lets_split/rev1/info.json]: thanks @fauxpark, corrected as of [c9255ef](https://github.com/qmk/qmk_firmware/pull/21855/commits/c9255ef41f81c6f656eb5b1dbab5df33d1045317)

**Comment by dunk2k** [comment]
> Might be more that I missed.

thanks @fauxpark, corrected as of [c9255ef](https://github.com/qmk/qmk_firmware/pull/21855/commits/c9255ef41f81c6f656eb5b1dbab5df33d1045317)

**Comment by fauxpark** [comment]
`handwired/arrow_pad:pad_21` seems to be failing, the rest look unrelated.

**Comment by dunk2k** [comment]
> `handwired/arrow_pad:pad_21` seems to be failing, the rest look unrelated.

I'm glad you've picked up on that one! Thankfully I've not touched it but having looked at it many moons ago, based on it's contents it's a keymap that really should either be it's own keyboard or a layout within `handwired/arrow_pad`.

I've compiled `pad21` keymap from `master` branch and it also fails; confirming this PR hasn't introduced a bug/error.

**Comment by dunk2k** [comment]
Applied fix for `handwired/arrow_pad:pad_21`, though in fairness it really could do with being addressed properly in a separate PR.

**Pull request closed without merging**